video_frame_metadata_registry.src.html

```diff
@@ -77,6 +77,10 @@
     <td>backgroundBlur</td>
     <td>[Background blur effect status](https://w3c.github.io/mediacapture-extensions/#background-blur-effect-status)</td>
   </tr>
+  <tr>
+    <td>backgroundSegmentationMask</td>
+    <td>[Background segmentation mask](https://github.com/w3c/mediacapture-extensions/pull/142)</td>
```

Comment:
https://github.com/w3c/mediacapture-extensions/pull/142 should eventually end up to be https://<span></span>w3c.github.io/mediacapture-extensions/#background-segmentation-mask (the section does not exist yet) and a preview is meanwhile accessible at  https://eehakkin.github.io/intel-w3c-mediacapture-extensions/#background-segmentation-mask.